nlp_primitives/tests/test_number_of_common_words.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+import pandas as pd
+
+from ..number_of_common_words import NumberOfCommonWords
+from .test_utils import PrimitiveT, find_applicable_primitives, valid_dfs
+
+
+class TestNumberOfCommonWords(PrimitiveT):
+    primitive = NumberOfCommonWords
+    test_word_bank = {'the', 'of', 'to', 'and', 'a', 'in', 'is', 'it', 'you', 'that'}
```

Comment:
Can we make this simpler and only include the values that we are matching on in the tests? That would make it a little easier to determine the expected results manually.